app/views/shared/_flash.html.erb

```diff
@@ -1,6 +1,7 @@
-<% flash.each do |key, message| %>
-  <div class="<%= flash_class(key) %> alert-dismissible" role="alert">
-    <a href="#" class="close" data-dismiss="alert" aria-label="close" style="text-decoration: none;"><%= fa_icon('times') %></a>
-    <%= message %>
-  </div>
+<%= turbo_frame_tag "flash" do %>
```

Comment:
In case we run the flash twice, we do want it to replace the second time as well. If we don't include the `turbo_frame_tag` in here it wouldn't reload again after the first flash